scripts/Engineering/gui/engineering_diffraction/tabs/focus/model.py

```diff
@@ -154,19 +157,53 @@ def _save_focused_output_files_as_nexus(self, instrument, sample_path, bank, sam
                 self._generate_output_file_name(instrument, sample_path, bank, ".nxs"))
             SaveNexus(InputWorkspace=sample_workspace, Filename=nexus_output_path)
 
-    def _save_focused_output_files_as_xye(self, instrument, sample_path, bank, sample_workspace,
-                                          rb_num):
+    def _save_focused_output_files_as_topas_xye(self, instrument, sample_path, bank,
+                                                sample_workspace, rb_num):
         xye_output_path = path.join(
             path_handling.get_output_path(), "Focus",
-            self._generate_output_file_name(instrument, sample_path, bank, ".dat"))
-        SaveFocusedXYE(InputWorkspace=sample_workspace, Filename=xye_output_path, SplitFiles=False)
+            self._generate_output_file_name(instrument, sample_path, bank, ".abc"))
+        SaveFocusedXYE(InputWorkspace=sample_workspace,
+                       Filename=xye_output_path,
+                       SplitFiles=False,
+                       Format="TOPAS")
         if rb_num:
             xye_output_path = path.join(
                 path_handling.get_output_path(), "User", rb_num, "Focus",
-                self._generate_output_file_name(instrument, sample_path, bank, ".dat"))
+                self._generate_output_file_name(instrument, sample_path, bank, ".abc"))
             SaveFocusedXYE(InputWorkspace=sample_workspace,
                            Filename=xye_output_path,
-                           SplitFiles=False)
+                           SplitFiles=False,
+                           Format="TOPAS")
+
+    @staticmethod
+    def _output_sample_logs(instrument, run_number, workspace, rb_num):
+        def write_to_file():
+            with open(output_path, "w", newline="") as logfile:
+                writer = csv.writer(f, ["Sample Log", "Avg Value"])
```

Comment:
Sorry, was thinking at the time that I should have commented both - but this also needs to be changed to logfile 
```suggestion
                writer = csv.writer(logfile, ["Sample Log", "Avg Value"])
```

Reply:
Yep, my bad. Should've checked. Fixed now.